### ISSUE TITLE
make a figma design and developed profile page

### ISSUE THREAD

**Issue by TejasNasre** [nexmeet/nexmeet#12]
build profile page with dashboards and other things..

profile will have : 
- user img
- user email
- user name
- section for organize event
- section to make their own community
- show all event hosted and participated

**Comment by NOOB-3301** [comment]
i want to work on it . Can you please assign it to me under swoc

**Comment by tejasnasre** [comment]
Yeah you can work but ishad also working on this so connect with him as well

**Comment by NOOB-3301** [comment]
i thought it is open, no one is assigned here , okey then i will message him.

> Yeah you can work but ishad also working on this so connect with him as well

**Comment by tejasnasre** [comment]
@NOOB-3301 why you mention this issue here

**Comment by NOOB-3301** [comment]
> @NOOB-3301 why you mention this issue here

i may maybe mistakenly mention it here . i just want to know if ishad is working thats all, sorry about that

**Comment by tejasnasre** [comment]
Ok
